project/Settings.scala

```diff
@@ -25,30 +25,49 @@ object Settings {
     https://github.com/sbt/sbt-assembly/issues/69
     https://github.com/scala/pickling/issues/10
 
-    Other fancy flags from
-
-    http://blog.threatstack.com/useful-scalac-options-for-better-scala-development-part-1
-
-    and
-
-    https://tpolecat.github.io/2014/04/11/scalac-flags.html
+    Other fancy flags from https://tpolecat.github.io/2017/04/25/scalac-flags.html.
 
+    The following aren't used (yet), and in general are an exercise in pain for 2.12 with Cromwell. They'd
+    certainly be nice to have, but params causes a world of hurt and patvars is just going to be a big time sink.
```

Comment:
holy moly, it'll be a time sink just understanding all the ones that did get turned on 😄